src/Class.js

```diff
@@ -1,10 +1,10 @@
 var WebJack = {};
 
-(function(exports){
```

Comment:
I just commented it out temporarily because it gave me an error (I think due to the assignment to 'this') and I didn't understand what its purpose is.